app/views/questionnaire.py

```diff
@@ -304,9 +306,18 @@ def get_view_submission(schema, eq_id, form_type):  # pylint: disable=unused-arg
             metadata_context = build_metadata_context_for_survey_completed(session_data)
 
             pepper = current_app.eq['secret_store'].get_secret_by_name('EQ_SERVER_SIDE_STORAGE_ENCRYPTION_USER_PEPPER')
+
             encrypter = StorageEncryption(current_user.user_id, current_user.user_ik, pepper)
+            submitted_data = encrypter.decrypt_data(submitted_data.data)
+
+            # for backwards compatibility
+            # submitted data used to be base64 encoded before encryption
+            try:
+                submitted_data = base64url_decode(submitted_data.decode()).decode()
```

Comment:
this can be removed after deployment too.

Reply:
Issue raised to track the removal - https://github.com/ONSdigital/eq-survey-runner/issues/1898